### PR TITLE
saap-196 adding cloud provider docs for GCP

### PR DESCRIPTION
**Pull request by TehreemNisa**
saap-196 adding cloud provider docs for GCP

### CODE REVIEW

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-202-600573c4`

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-202-3500966f`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-202-5d2be0da`

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-202-2746ee3f`

File: content/sre/cloud-provider/google.md [hunk 1]
```diff
@@ -1,3 +1,26 @@
...
+  | Service accounts | 5 |
+  | Firewall Rules | 11|
+  | Forwarding Rules | 2|
+  | Routers | 1|
+  | Routes | 2|
+  | Subnetworks | 2|
+  | Firewall Rules | 11|
+  | Persistent Disk SSD (GB)| 896|
+  | Target Pools | 3|
+  | Machine Specifications | 6 machines of 8x32x120G |
```
Comment (usamaahmadkhan) [inline content/sre/cloud-provider/google.md]: 9 machines of 8x32x120G

Comment (usamaahmadkhan) [reply content/sre/cloud-provider/google.md]: Plz update this for aws openstack etc.

Comment (TehreemNisa) [reply content/sre/cloud-provider/google.md]: updated

**Review by usamaahmadkhan — APPROVED** [review]

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-202-5543a9a8`

**Comment by stakater-user** [comment]
@usamaahmadkhan Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:v0.0.143`

**Pull request merged**